app/protected/solve/page.tsx

```
@@ -0,0 +1,205 @@
+        };
+
+        setChatHistory((prev) => ({
+            messages: [...prev.messages, userMessage],
+        }));
+
+        setIsLoading(true);
+        try {
+            /* TO-DO: implement API call to get reponse for chatbot */
+        } catch (error) {
+            console.log(error);
+        } finally {
```

Biome (2.1.2)

[error] This code is unreachable



(lint/correctness/noUnreachable)